vault/expiration.go

```diff
@@ -114,6 +116,58 @@ func (c *Core) stopExpiration() error {
 	return nil
 }
 
+// Tidy cleans up stale leases which are associated with invalid tokens
+func (m *ExpirationManager) Tidy() error {
+	var tidyErrors *multierror.Error
+
+	tidyFunc := func(leaseID string) {
```

Comment:
Please add some progress logging so people can be aware things are still happening.